### PR TITLE
Initial files

### PR DESCRIPTION
**Pull request by 2bleO**
In this we:

Configured linters
Create index.html file
Create style.css file
Create gitignore file
Ignored tst.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
...
+- HTML
+- CSS
+
+## Author
+
+👤 **Onesis Olea**
+
+- GitHub: [@2bleo](https://github.com/2bleO)
+- Twitter: [@OleaOnesis](https://twitter.com/OleaOnesis)
+- LinkedIn: [Onesis Olea](www.linkedin.com/in/onesis-olea)
```
Comment (descholar-ceo) [inline README.md]: - [ ] [Optional] This link is broken, would you fix it?

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+
+## Acknowledgments
+
+* [GitHub](https://www.github.com)
+* [Microverse](https://microverse.org)
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (descholar-ceo) [inline README.md]: - [ ] [Optional] This link is broken as well, would you fix it?

File: test.md [hunk 3]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (descholar-ceo) [inline test.md]: - [ ] As it is said, this file should be ignored by git, this means that it shouldn't appear here in your repo, removing it, you can use `git rm -f test.md` and writing it on gitignore, you should remove the starting slash, and then commit changes

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: 
## Hi @2bleO  :handshake: :handshake: :handshake:  , 

Good job so far!
We are so excited to have you here at Microverse, with your first project here!!! :smile: :smile: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :high_brightness: :heavy_check_mark: 
:+1: The linters are set and they are all green :heavy_check_mark: 
:+1: The repository looks professional :heavy_check_mark:
:+1: Github flow was correctly :heavy_check_mark:

## Required Changes ♻️
- [ ] The test.md file is still present from your repo, kindly take notes on the comment I put on the test.md file, and address the issue.

## Optional suggestions :yellow_circle: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] In order to make your repo more professional, you need to make sure that all info presented on your readme file is accurate, I suggest you fix all issues I commented in your readme file

- [x] You have meaningful commit messages, but note that they need to be capitalized, this means that all of commit messages should have their first letter capitalized in that case it will be looking professional. [this article](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) can help you learn about a professional commit message

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 2bleO** [comment]
Hi, @descholar-ceo 
Thank you for your feedback. I appreciate your efforts in helping us with code reviews.
(I think) I made the required changes.

**Review by jubaan — APPROVED** [review]
Hi @2bleO , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**